src/opm/input/eclipse/share/keywords/000_Eclipse100/N/NODEPROP

```diff
@@ -3,6 +3,8 @@
   "sections": [
     "SCHEDULE"
   ],
+  "requires": ["NETWORK", "BRANPROP"],
```

Comment:
I was not really aware of this feature, have we had it all the time?!?

Reply:
> have we had [`requires`/`prohibits`] all the time?!?

The feature was introduced in October 2020 (PR #1996), but it's not particularly widely deployed.  Most of its usage is in the table input code, and especially in the saturation functions (`SOF3` etc.) to reject decks using saturation function tables from both families.